docs/testing_configuration_tests.rst

```diff
@@ -91,13 +91,35 @@ mirroring the Ansible configuration. Prior to the reorganization of
 the Ansible layout, the tests are rather tightly coupled to hosts. The
 layout of config tests is therefore subject to change.
 
-Config Testing Strategy
------------------------
+Running the CI Staging Environment
+----------------------------------
 
-The config tests currently emphasize testing implementation rather than
-functionality. This was a temporary measure to increase the testing
-baseline for validating the Ansible provisioning flow, which aided in migrating
-to a current version of Ansible (v2+). Now that the Ansible version is current,
-the config tests can be improved to validate behavior, such as confirming
-ports are blocked via external network calls, rather than simply checking
-that the iptables rules are formatted as expected.
+The staging environment can also run via CI, running in GCE. These tests are
+run every night or if you push to a branch that starts with ``stg-``. Currently
+this can only be done by members of the ``freedomofpress`` GitHub organization,
+please ask if you'd like someone to run the tests for you.
```

Comment:
Is it well understood where people should ask for this? Is it worth including an email/link to the github issues page where people should submit this request?

Reply:
I'll say "Please ask in your PR if..."